backend-truffle/contracts/DoorManager.sol

```diff
@@ -1,77 +1,61 @@
-//Meet.inc >>> ĐOor-Manager <<<
-//The ĐOor-Manager is a factory that allows event managers to deploy their own Door Event.
-//ĐOor opens doors at events that are anchored on the Ethereum blockchain.
-//
-//Last update: 10.11.2019
-
 pragma solidity ^0.5.0;
 
 import "@openzeppelin/contracts/ownership/Ownable.sol";
 import "@openzeppelin/contracts/lifecycle/Pausable.sol";
-import "./DoorEvent.sol";
+import "./Door.sol";
```

Comment:
I want to distinguish between 
- Door Event
- Door Meetup
- Door Dinner
....etc